source.lua

```diff
@@ -2449,6 +2449,7 @@ end
 
 function RayfieldLibrary:SetVisibility(visibility: boolean)
 	setVisibility(visibility, false)
+	return Hidden

```

Comment:
Arguably, I would prefer if instead of returning it from a setter function, a new function be made called `GetVisibility` to maintain more clear code.